backend/src/test/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/external/VerkuendungControllerTest.java

```diff
@@ -0,0 +1,159 @@
+package de.bund.digitalservice.ris.norms.adapter.input.restapi.controller.external;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import de.bund.digitalservice.ris.norms.adapter.input.restapi.controller.SecurelessControllerTest;
+import de.bund.digitalservice.ris.norms.application.port.input.LoadNormendokumentationspacketProcessingStatusUseCase;
+import de.bund.digitalservice.ris.norms.application.port.input.StoreNormendokumentationspaketUseCase;
+import de.bund.digitalservice.ris.norms.domain.entity.VerkuendungImportProcess;
+import de.bund.digitalservice.ris.norms.domain.entity.VerkuendungImportProcessDetail;
+import java.io.InputStream;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SecurelessControllerTest(VerkuendungController.class)
+class VerkuendungControllerTest {
```

Comment:
The test for import not found is missing, whether in the unit test or in the integration one, don't really care :)